plugins/LilyPad/Config.h

```diff
@@ -21,13 +21,13 @@
 extern u8 ps2e;
 
 enum PadType {
+    // Changing the number of PadTypes requires numPadTypes in InputManager.h to be modified as well.
```

Comment:
Maybe turtleli meant something like this?

Currently
> DisabledPad = 0,
     Dualshock2Pad = 1,
     GuitarPad = 2,
     PopnPad = 3

If you add a new enum called ``Pad Count`` then,

>> DisabledPad = 0,
     Dualshock2Pad = 1,
     GuitarPad = 2,
     PopnPad = 3,
     PadCount = 4 // Counts the total number of pads and is flexible as it adjusts itself when more enums are added